### ISSUE TITLE
[Bug]: Maintenance Room Spaces Echo Station

### ISSUE THREAD

**Issue by MarkusLarsson421** [BeeStation/BeeStation-Hornet#11513]
### Round Date

2024/09/17

### Describe the bug

I was told that a maintenance room, most likely sk_rdm149_cratewindow.dmm, which includes space will cause issues on Echo by spacing nearby areas. Unsure if it spaces the floor below the maintenance room or not. Outer space should not exist on a planet. 

![image](https://github.com/user-attachments/assets/c07ad06c-392a-4c2f-86e3-fc9aa2f7bcc6)

### What did you expect?

I expected outer space to not be on a planet.

### Reproducible steps

1. RNG.

**Comment by EvilDragonfiend** [comment]
oh, that is an interesting rng bug case. good catch.

**Comment by MarkusLarsson421** [comment]
Yeah, I am unsure if it should be removed or an exclusion list should be added to Echo Station.
